### PR TITLE
new host ui details

### PR DESCRIPTION
**Pull request by tstrych**
This is the base for new UI host details. It can be further improved by anyone. I think this is a good concept. 

This PR is introducing the first 2 Cards in Host details Overview, and also the system to move from one tab to another tab.

### CODE REVIEW

**Comment by tstrych** [comment]
depends on [#9399](https://github.com/SatelliteQE/robottelo/pull/9399)

File: airgun/widgets.py [hunk 1]
```diff
@@ -949,7 +949,7 @@ def add(self, value):
         :param value: dict with format {'name': str, 'value': str}
         """
         self.add_new_value.click()
-        new_row = self.__getitem__(-1)
+        new_row = self.__getitem__(0)
```
Comment (tstrych) [inline airgun/widgets.py:952]: new rows are now adding from the top, not from the bottom

File: airgun/views/host_new.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                labels.extend([None] * (len(values) - len(labels)))
+            else:
+                values.extend([None] * (len(labels) - len(values)))
+        for key, value in zip(labels, values):
+            if not key or not value:
+                continue
+            value = self.browser.text(value)
+            key = self.browser.text(key).replace(' ', '_').lower()
+            items[key] = value
+        return items
```
Comment (omkarkhatavkar) [inline airgun/views/host_new.py:44]:
```python
        if key and value:
            value = self.browser.text(value)
            key = self.browser.text(key).replace(' ', '_').lower()
            items[key] = value
``` This also should work

File: airgun/widgets.py (hunk 1 above)
Comment (omkarkhatavkar) [reply airgun/widgets.py:952]: Can you test this that it is not breaking other places this code has been used

**Comment by latran** [comment]
@tstrych I've asked UI to add ouia-ID so there's static IDs.  You can put this PR in draft til they merge the changes.

**Comment by tstrych** [comment]
@latran this is working, and their fix can take time, and I implement the usage of new host view details for old UI tests where it's necessary to navigate from new details to the old one

I can change locators afterwards

**Review by omkarkhatavkar — APPROVED** [review]
ACK

**Review by jyejare — APPROVED** [review]

**Pull request merged**